### ISSUE TITLE
Cannot update boolean column using graphql mutation

### ISSUE THREAD

**Issue by artalat** [supabase/pg_graphql#162]
**Describe the bug**
When trying to update a boolean column via update mutation using the graphql API, server throws that following error:

```json
{
  "data": null,
  "errors": [
    {
      "message": "column \"completed\" is of type boolean but expression is of type text"
    }
  ]
}
```

**To Reproduce**

Using a typical todo example

Mutation:

```graphql
mutation UpdateTaskMutation(
  $id: UUID!
  $title: String
  $description: String
  $completed: Boolean
) {
  updatetasksCollection(
    set: { title: $title, description: $description, completed: $completed }
    filter: { id: { eq: $id } }
  ) {
    affectedCount
    records {
      id
      title
      description
      completed
    }
  }
}

```

Query Variables:

```json
{
  "id": "9def5b70-f0b0-443c-bb23-0643e90a6d2c",
  "title": "some title",
  "description": "a description",
  "completed": true
}
```

**Expected behavior**
To successfully update data in the database.

**Screenshots**
N/A

**Versions:**
Cloud hosted version

**Additional context**
N/A

**Comment by artalat** [comment]
> this was resolved in release `v0.2.1` in [this PR](https://github.com/supabase/pg_graphql/pull/154)
> 
> if you send me your supabase project ref I'll update your instance so you don't have to wait for the next release to receive the fix

Thanks, here it is `stzovznbknzbjaepwvpq`

**Comment by olirice** [comment]
thanks, updated
please let me know if you continue to see any issues with that query
